### DISCUSSION TITLE
How to deploy a new model based on triton and fastertransformer_backend?

### DISCUSSION THREAD

**Discussion by WangYizhang01** [triton-inference-server/fastertransformer_backend#104] [Q&A] [answered]
Hello, I am new to fastertransformer_backend and there are still many things that are not very clear to me. I have some questions to consult with you, mainly about how to deploy a new model.
Now fastertransformer_backend supports several models such as Bert and GPT. If I want to deploy a new model (not included in the Support matrix), such as AlphaFold, what work do I need to do?
Looking forward to your reply！

**Comment by byshiue** [comment, marked as answer]
FasterTransformer cannot parse the model architecture. So, for a new model, you may need to develop the model (different model architectures and cuda kernels) first, and then encapsulate it by the triton backend. Then, you can call it in triton.

**Reply by siddharth-mavani** [reply]
Can you please explain further on what you mean by "you may need to develop the model (different model architectures and cuda kernels) first". Secondly, how do I encapsulate it by the triton backend ?

Is there documentation available related to the above queries ?
